code/game/turfs/open/floor/plating/wasteplanet.dm

```diff
@@ -28,13 +32,40 @@
 	desc = "Corrupted steel."
 	icon_state = "plating_rust"
 
+/turf/open/floor/wood/waste
+	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/indestructible/hierophant/waste
+	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/indestructible/hierophant/two/waste
+	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+	light_color = LIGHT_COLOR_FLARE
+
+/turf/open/water/waste
+	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/floor/plating/grass/wasteplanet
+	icon_state = "junglegrass"
+	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
+/turf/open/floor/plating/dirt/old/waste
+	initial_gas_mix = WASTEPLANET_DEFAULT_ATMOS
+	planetary_atmos = TRUE
+
 //open turfs then open lits.
 
 /turf/open/floor/plating/wasteplanet/lit
 	light_range = 2
 	light_power = 0.2
 	light_color = LIGHT_COLOR_FLARE
 
+/turf/open/floor/plating/rust/wasteplanet/lit
```

Comment:
you need to scrub this out. the already defined path is /plating/wasteplanet/rust/lit